packs/feats/educate-allies.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p><strong>Trigger</strong> An ally benefiting from one of your composition spells is subject to an effect with the auditory, illusion, linguistic, sonic, or visual trait.</p>\n<hr />\n<p>You tweak the properties of your composition spell to convey a bit of your defensive knowledge. All allies affected by your composition spell gain your +1 circumstance bonus from @UUID[Compendium.pf2e.feats-srd.Item.Well-Versed] until the start of your next turn. Teaching your allies also bolsters your own skills; your personal circumstance bonus from Well-Versed also increases to +2 until the start of your next turn.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Educate Allies (Ally)]</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Educate Allies (Self)]</p>"
```

Comment:
Want to add this to the effect condensing tracking effect? https://github.com/foundryvtt/pf2e/issues/4144

Reply:
You got it. There is some Blood Magic stuff I did a few weeks ago I should add too.